multinode/storage/service.go

```diff
@@ -159,6 +163,10 @@ func (service *Service) TotalDiskSpace(ctx context.Context) (totalDiskSpace Disk
 	}
 
 	for _, node := range listNodes {
+		nodeinfo := service.nodes.FetchNodeInfo(ctx, node)
+		if nodeinfo.Status != nodes.StatusOnline {
+			continue
+		}
 		diskSpace, err := service.dialDiskSpace(ctx, node)
 		if err != nil {
```

Comment:
@ifraixedes we can also do one things if there is a err then we can just skip that. What so you think?

Reply:
I don't understand well your question.

What do you suggest to do if there is an error?

Reply:
```
monthly, err := service.getMonthly(ctx, node)
		if err != nil {
			continue
		}
``` 
in stead of 
```
monthly, err := service.getMonthly(ctx, node)
		if err != nil {
			if nodes.ErrNodeNotReachable.Has(err) {
				continue
			}

			return Monthly{}, Error.Wrap(err)
		}
``` We can just skip the nodes which have errors.

Reply:
I agree with skipping the nodes with errors as long as the status shows that they're not online. Also we shouldn't entirely ignore the errors but at least log them